Source/Engine/Visject/VisjectGraph.cpp

```diff
@@ -371,6 +371,19 @@ void VisjectExecutor::ProcessGroupMath(Box* box, Node* node, Value& value)
         if (value.Type.Type == VariantType::Enum)
             value.AsUint64 = value.AsUint64 | (uint64)tryGetValue(node->GetBox(1), Value::Zero);
         break;
+
+    case 48:
+    {
+        const float inVal  = tryGetValue(node->GetBox(0), node->Values[0]).AsFloat;
+        const Vector2 rangeA = tryGetValue(node->GetBox(1), node->Values[1]).AsVector2();
+        const Vector2 rangeB = tryGetValue(node->GetBox(2), node->Values[2]).AsVector2();
+
+        auto mapFunc = rangeB.X + (inVal - rangeA.X) * (rangeB.Y - rangeB.X) / (rangeA.Y - rangeA.X);
+        
+        // Clamp value?
+        value = node->Values[3].AsBool ? Math::Clamp(mapFunc, rangeB.X, rangeB.Y) : mapFunc;
```

Comment:
Instead of using `node->Values[3].AsBool` call `tryGetValue` like for other inputs to handle both default value and custom connected value.

Reply:
Applied